src/components/Card.js

```diff
@@ -15,22 +25,32 @@ export default class Card {
 
   //event listeners for cards
   _setEventListeners() {
-    this._likeButton.addEventListener("click", this._likeCard);
-    this._deleteButton.addEventListener("click", this._removeCard);
+    this._likeButton.addEventListener("click", () => {
+      this.handleLikeClick(this);
+      this._checkedIfLiked();
+    });
+    this._deleteButton.addEventListener("click", () => {
+      this.handleTrashClick(this);
+    });
     this._cardImage.addEventListener("click", this.handleImageClick);
   }
 
-  _likeCard(e) {
-    e.target.classList.toggle("card__like-icon_clicked");
-  }
+  _checkedIfLiked = () => {
+    if (this.isLiked) {
+      this._likeButton.classList.add("card__like-icon_clicked");
+    } else {
+      this._likeButton.classList.remove("card__like-icon_clicked");
+    }
+  };
 
-  _removeCard(e) {
-    const cardElement = e.target.closest(".card");
-    cardElement.remove();
-  }
+  _removeCard = () => {
```

Comment:
Needs correction: this method should be public as you should use it outside of a class